codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Exception.java

```diff
@@ -229,6 +251,19 @@ public int streamId() {
         }
     }
 
+    public static final class HeaderListSizeException extends StreamException {
+        private boolean decode;
+
+        HeaderListSizeException(int streamId, Http2Error error, String message, boolean decode) {
+            super(streamId, error, message);
+            this.decode = decode;
+        }
+
+        boolean duringDecode() {
```

Comment:
this is not used anywhere. Do we think it will be useful and if so should we make it `public`?